dynawo/sources/Modeler/DataInterface/DYNDataInterfaceIIDMCommon.cpp

```diff
@@ -0,0 +1,128 @@
+//
+// Copyright (c) 2022, RTE (http://www.rte-france.com)
+// See AUTHORS.txt
+// All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at http://mozilla.org/MPL/2.0/.
+// SPDX-License-Identifier: MPL-2.0
+//
+// This file is part of Dynawo, an hybrid C++/Modelica open source time domain
```

Comment:
Not the latest description of dynawo
`// This file is part of Dynawo, an hybrid C++/Modelica open source suite of simulation tools

// for power systems.`

Reply:
done